akka-http-core/src/main/resources/reference.conf

```diff
@@ -431,6 +443,18 @@ akka.http {
 
       # Enable verbose debug logging for all ingoing and outgoing frames
       log-frames = false
+
+      # When there is no data transmitted, send a HTTP/2 ping frame with this interval to make sure the connection is
+      # kept alive, 0s disables sending ping frames. Must be full seconds.
+      keepalive-time = 0s
+
+      # Fail the connection if a ping frame is sent and does not come back from the server within this timeout.
+      # Must be full seconds and less than the ping interval.
+      keepalive-timeout = 5s
```

Comment:
I'd say the default is too low. Too easy to be stuck behind data that was just send at the same time as the ping was sent.

Maybe just `ping-ack-timeout`.

Reply:
I think it should perhaps not have a default at all since it needs to be lower than the interval with the current implementation, and the interval does not have a default but is set by the user to enable pinging.